scripts/gha-shards-readme.md

```diff
@@ -0,0 +1,35 @@
+# Unit tests sharding strategy
+
+#### background
+Unit tests are run in parallel (in GitHub Actions matrices) using the sharding strategy specified in unit-test-shards.json
+We've divided the top level modules into multiple shards to achieve better parallelism.
+The configuration in unit-test-shards.json specifies the shard name as key for each shard and the value contains an object
+with django settings for each module and paths for submodules to test for example:
+```json
+{
+    "lms-1": {
+        "paths": ["lms/djangoapps/course_api", ...],
+        "settings": "lms.envs.test",
+    }
+    .
+    .
+    .
+}
+```
+`common` and `openedx` modules are tested with both `lms` and `cms` settings that's why there are shards with same `opendex`
+submodules but with different django settings.
+For more details on sharding strategy please refer to this section on [sharding](https://openedx.atlassian.net/wiki/spaces/AT/pages/3235971586/edx-platfrom+unit+tests+migration+from+Jenkins+to+Github+Actions#Motivation-for-sharding-manually)
+
+#### Unit tests count check is failing
+There's a check in place that makes sure that all the unit tests under edx-platform modules are specified in `unit-test-shards.json`
+If there's a mismatch between the number of unit tests collected from `unit-test-shards.json` and the number of unit tests collected
+against the entire codebase the check will fail.
+You'd have to update the `unit-test-shards.json` file manually to fix this.
+
+##### How to fix
+- If you've added a new django app to the codebase, and you want to add it to the unit tests you need to add it to the `unit-test-shards.json`
```

Comment:
How do I decide which shard to add it to?

Reply:
https://openedx.atlassian.net/wiki/spaces/AT/pages/3235971586/edx-platfrom+unit+tests+migration+from+Jenkins+to+Github+Actions#Where-should-I-place-my-new-Django-app

Reply:
I would add this under "How to fix":
```
##### How to fix
You will need to understand details of our sharding strategy, documented in https://openedx.atlassian.net/wiki/spaces/AT/pages/3235971586/edx-platfrom+unit+tests+migration+from+Jenkins+to+Github+Actions. 
- If you've added a new Django app, you need to add it to `unit-test-shards.json`.  The wiki page explains how to choose the right shard.
- ...
```